android/app/src/main/res/drawable-v21/launch_background.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could these not be included with this PR, they seem unrelated

Reply:
I didn't change them, It changed itself